Automattic/jetpack
# Fix Pay via Paypal block: currency symbol appears after the price in editor

fixes #11897

#### Changes proposed in this Pull Request:

* Simple payments block now uses `Intl.NumberFormat()` to format currency display, if available

#### Jetpack product discussion

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* On master branch, create a new post.
* Add a 'Pay via Paypal' block and fill out the information.  Save as draft.
* Unfocus the block.  In preview mode, instead of "$5.00", it will say "5.00 $".
* Switch to this branch and reload the post.
* The block should upgrade correctly and now display $5.00.
* Other currencies and locales should be supported as well.

![2020-10-06_14-25](https://user-images.githubusercontent.com/937354/95254008-e15f7700-07e4-11eb-870c-44ff5bd5c951.png)

( To test on my sandbox, I used `yarn build-extensions --watch` and then rsynced the `./_inc/blocks` folder over).

#### Proposed changelog entry for your changes:

* Simple payments block now uses `Intl.NumberFormat()` to format currency display, if available
